FF1Lib/Hacks.cs

```diff
@@ -2557,5 +2557,40 @@ public void NoItemMagic(Flags flags)
 
 			WriteText(itemnames, FF1Rom.ItemTextPointerOffset, FF1Rom.ItemTextPointerBase, FF1Rom.ItemTextOffset, FF1Rom.UnusedGoldItems);
 		}
+
+		public void HackMinimap(OverworldMap map) {
+		    var compresedMap = map.GetCompressedMapRows();

```

Comment:
compres**s**edMap?

Reply:
You only use it once, can probably just chuck it.

Reply:
That variable is gone now